websocket/src/websocket.h

```diff
@@ -35,6 +39,7 @@ namespace dmWebsocket
 
     enum State
     {
+        STATE_CONNECT,
```

Comment:
I would still change it to "create" or something. Or change the other one.

Reply:
Good suggestion. Changed it!